<?xml version="1.0" encoding="UTF-8"?>
<!-- Vị trí: /ui/widgets/downloadoptionswidget.ui -->
<!-- Phiên bản: 1.2 (Hoàn thiện UX) -->
<ui version="4.0">
 <class>DownloadOptionsWidget</class>
 <widget class="QWidget" name="DownloadOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="downloadOptionsGroupBox">
     <property name="title">
      <string>Tùy chọn tải</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Thumbnail:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="thumbnailComboBox"/>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="downloadPlaylistCheckBox">
          <property name="text">
           <string>Tải playlist</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QCheckBox" name="cutVideoCheckBox">
          <property name="text">
           <string>Cắt video</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLineEdit" name="cutTimeLineEdit">
          <property name="placeholderText">
           <string>HH:MM:SS-HH:MM:SS</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Phụ đề:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="subtitleComboBox"/>
        </item>
        <item row="1" column="2">
         <widget class="QCheckBox" name="writeDescriptionCheckBox">
          <property name="text">
           <string>Tải mô tả</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="useCustomCommandCheckBox">
          <property name="text">
           <string>Nhập lệnh tùy chỉnh</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="customCommandComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="clearCustomCommandButton">
          <property name="text">
           <string>X</string>
          </property>
          <property name="styleSheet">
           <string>color: red; font-weight: bold;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>

